Source/Core/PolygonGeometryLibrary.js

```diff
@@ -191,6 +257,57 @@ PolygonGeometryLibrary.subdivideLine = function (p0, p1, minDistance, result) {
   return positions;
 };
 
+/**
+ * Subdivides texture coordinates based on the subdivision of the associated world positions using a rhumb line.
+ *
+ * @param {Cartesian2} t0 First texture coordinate.
+ * @param {Cartesian2} t1 Second texture coordinate.
+ * @param {Ellipsoid} ellipsoid The ellipsoid.
+ * @param {Cartesian3} p0 First world position.
+ * @param {Cartesian3} p1 Second world position.
+ * @param {Number} minDistance Minimum distance for a segment.
+ * @param {Array<Cartesian2>} result The subdivided texture coordinates.
+ *
+ * @private
+ */
+PolygonGeometryLibrary.subdivideTexcoordRhumbLine = function (
+  t0,
+  t1,
+  ellipsoid,
+  p0,
+  p1,
+  minDistance,
+  result
+) {
+  // Compute the surface distance.
+  const c0 = ellipsoid.cartesianToCartographic(p0, scratchCartographic0);
+  const c1 = ellipsoid.cartesianToCartographic(p1, scratchCartographic1);
+  const rhumb = new EllipsoidRhumbLine(c0, c1, ellipsoid);
```

Comment:
This could be a scratch variable instead. Use `setEndPoints` to reset the start/end.

Reply:
Updated to use scratch variable.